src/components/storefront/Item/Item.js

```diff
@@ -8,9 +8,11 @@ const Item = ({ id, name, image, price }) => {
   const { path } = useRouteMatch();
 
   return (
-    <VStack>
-      <Link as={PreserveQueryParamsLink} to={`${path}/products/${id}`}>
-        <Image border="1px solid black" w={80} h={80} src={image} alt={name} />
+    <VStack textAlign="center">
+      <Link w="100%" as={PreserveQueryParamsLink} to={`${path}/products/${id}`} bg="white">
+        <AspectRatio maxW="400px" ratio={1}>
```

Comment:
whoa this is so cool I never knew this existed

Reply:
LOL yeah I swear I just search Chakra's docs and they have something for everything it's lit